go/vt/vtctl/reparentutil/replication.go

```diff
@@ -147,7 +147,9 @@ func ReplicaWasRunning(stopStatus *replicationdatapb.StopReplicationStatus) (boo
 		return false, vterrors.Errorf(vtrpc.Code_INVALID_ARGUMENT, "could not determine Before state of StopReplicationStatus %v", stopStatus)
 	}
 
-	return stopStatus.Before.IoThreadRunning || stopStatus.Before.SqlThreadRunning, nil
+	ioThreadRunning := stopStatus.Before.IoThreadRunningState == "Yes" || stopStatus.Before.IoThreadRunningState == "Connecting"
```

Comment:
Isn't this just `stopStatus.Before.IOThreadRunning()`?

Reply:
It's not because this is the proto generated struct (whose methods are auto-generated so I can't add that method there)